experimental/web_dashboard/lib/src/app.dart

```diff
@@ -51,10 +51,10 @@ class DashboardApp extends StatefulWidget {
         super(key: key);
 
   @override
-  _DashboardAppState createState() => _DashboardAppState();
+  DashboardAppState createState() => DashboardAppState();
```

Comment:
Instead of making the `_DashboardAppState ` objects public, I'd change the return value of the `createState ()` accessor from `_DashboardAppState` to `State<Dashboard>`

Reply:
Updated all instances in this PR